src/modules/authentacation/components/Login/Login.tsx

```diff
@@ -1,7 +1,118 @@
-// import React from 'react'
+import { useContext, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { FaRegEyeSlash, FaSpinner } from 'react-icons/fa';
+import { IoEyeOutline } from 'react-icons/io5';
+import { Link, useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { AuthContext } from '../../../../context/AuthContext/AuthContext';
+import {
+  axiosInstance,
+  USERS_URLS,
+} from '../../../../services/apisUrls/apisUrls';
+import {
+  EMAIL_VALIDATION,
+  PASSWORD_VALIDATION,
+} from '../../../../services/validation/validation';
+
+interface loginData {
+  email: string;
+  password: string;
+}
 
 export default function Login() {
+  const [isPasswordVisable, setIsPasswordVisable] = useState(false);
+  const navigate = useNavigate();
+  const { saveLoginData} = useContext(AuthContext);
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+  } = useForm();
+
+  const onSubmit = async (data: loginData) => {
+    // console.log(data);
+    try {
+      const response = await axiosInstance.post(USERS_URLS.LOGIN, data);
```

Comment:
We can provide the type of the response to the axios method to be able to get auto completion and avoid accessing incorrect fields `post<LoginResponse>`